main/nginx/APKBUILD

```diff
@@ -94,7 +94,7 @@ _add_module "http-fancyindex" "v0.4.3" "https://github.com/aperezdc/ngx-fancyind
 _add_module "http-headers-more" "v0.33" "https://github.com/openresty/headers-more-nginx-module"
 _http_headers_more_so="ngx_http_headers_more_filter_module.so"
 
-_add_module "http-lua" "v0.10.13" "https://github.com/openresty/lua-nginx-module"
+_add_module "http-lua" "v0.10.14" "https://github.com/openresty/lua-nginx-module"
```

Comment:
you should increment `pkgrel` also it may need fresher tests like https://github.com/alpinelinux/aports/pull/6853

Reply:
Done.

- Incremented `pkgrel`
- Upgraded `neginx-tests` to HG rev: `4bc1490cedbc`

Reply:
The build seems to fail, nevertheless, I'm unable to reproduce that same build failure locally.
Got a hint for me here @andypost?